golf_players.py

```diff
@@ -0,0 +1,51 @@
+def choice_of_option():
+    while True:
+        print('If you wanna write a player\'s result press 1')
+        print('If you wanna leave the program press 2')
+        try:
```

Comment:
Extra task: in this program you have two functions that do similar thing - they get user's input and check whether it's a number. Maybe we can write such code only once, and reuse it?